site/ja/tutorials/keras/basic_classification.ipynb

```diff
@@ -797,7 +796,7 @@
         "    thisplot = plt.bar(range(10), predictions_array, color=\"#777777\")\n",
         "    plt.ylim([0, 1])\n",
         "    predicted_label = np.argmax(predictions_array)\n",
-        "\n",
+        "    \n",
```

Comment:
I also noticed that this sample code doesn't work if I just copy & paste because of unaligned indent.

Reply:
Did you paste these code on python command prompt?

These example codes are intend to be executed on jupyter notebooks.
Blank lines are ignored before execution except in python interpreter command prompt.

Although lint program like pycodestyle/pep8 warns with "W293 blank line contains whitespace", it seems good for compatibility to add indent on blank lines.

I'm not sure which is better.  There are argument on this topic.
https://stackoverflow.com/questions/2727988/python-indentation-in-empty-lines

Reply:
I'm in the  "if the tooling doesn't do it automatically, don't spend too much time on it" camp.